src/witness_generator/machines/fixed_lookup_machine.rs

```diff
@@ -76,49 +78,100 @@ impl FixedLookup {
         left: &[Result<AffineExpression, EvalError>],
         right: &SelectedExpressions,
     ) -> EvalResult {
```

Comment:
This function returns new knowledge but somehow witness generation still runs forever 🤔

Reply:
Ok it was just slow lol